test/scenarios/el92-src@log-scan.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-source
+    launch_vm host1
+}
+
+scenario_remove_vms() {
+    remove_vm host1
+}
+
+scenario_run_tests() {
+    run_tests host1 suites/logscan/
```

Comment:
These tests look specifically for graceful start of MicroShift. When doing restarts you disrupt the pods and the apiserver in such a way that errors will happen, but shouldnt on a fresh reboot.
These tests also look at the journal without knowing where to stop since the test execution order is randomized. If there have been no restarts it works, if there were any it may fail.

Reply:
Can we avoid adding another VM? 
I mean, if we're looking for clean logs, can we deleted them?

P.S. It's getting very tight with the number of VMs

Reply:
I am thinking of doing something different, which is to wipe the whole microshift deployment as part of this test. This is disruptive with what was running before, but getting a clean log start from an already running system is somehow complex/flaky by nature. Would that be ok? I would do it as part of a different PR though.